v1/tasks/admin.py

```diff
@@ -3,5 +3,4 @@
 
 from .models import Task
 
-
```

Comment:
i'm used to have 2 lines after imports block ... always ... but there is no strict policy in any standard for this particular case